var/spack/repos/builtin/packages/halide/package.py

```diff
@@ -14,6 +14,7 @@ class Halide(CMakePackage, PythonExtension):
     git = "https://github.com/halide/Halide.git"
     maintainers = ["wraith1995"]
```

Comment:
```suggestion
    maintainers("wraith1995")
```
This appends you to the list so I still get pinged.